package.json

```diff
@@ -2,38 +2,51 @@
   "name": "molotov",
   "description": "Molotov: (Mix/Plug)-in loader for explosive composeability",
   "main": "molotov.js",
+  "version": "0.0.0-development",
   "directories": {
     "example": "examples"
   },
   "dependencies": {
-    "callsite": "^1.0.0",
```

Comment:
![yas](https://m.popkey.co/f961d6/dZOw6.gif)

Reply:
So fab. Could i in any way ever be so fab? no probably not.